internal/infrastructure/price-feeder/repository.go

```diff
@@ -0,0 +1,25 @@
+package pricefeederinfra
+
+import (
+	"context"
+)
+
+type PriceFeedRepository interface {
+	// AddPriceFeed adds a new price feed to the repository.
+	AddPriceFeed(ctx context.Context, priceFeed PriceFeed) error
+	// GetPriceFeed returns the price feed with the given ID.
+	GetPriceFeed(ctx context.Context, id string) (*PriceFeed, error)
+	// GetPriceFeedsByMarket returns all price feed for a given market.
+	GetPriceFeedsByMarket(
```

Comment:
ah! you used a naming that i do really like here but you basically reverted in the portable interface by specifying GetPriceFeed and GetPriceFeedForFeedID.

This can cause confusion so let's use this naming also in `ports.PriceFeeded` as long as we still need a way to get price feeds by market.

Reply:
Done